export const getTransactions = () => {
    try {
        
    }
    catch(error) {
        console.error(error)
    }
}